### PR TITLE
replace deprecated constructors with builders.

### PR DESCRIPTION
**Pull request by albertshau**

### CODE REVIEW

File: mongodb-plugins/src/test/java/co/cask/hydrator/plugin/test/MongoDBTest.java [hunk 1]
```diff
@@ -228,7 +232,10 @@ public void testMongoDBSink() throws Exception {
         .put(MongoDBBatchSink.Properties.CONNECTION_STRING,
              String.format("mongodb://localhost:%d/%s.%s",
                            mongoPort, MONGO_DB, MONGO_SINK_COLLECTIONS)).build()));
-    ETLBatchConfig etlConfig = new ETLBatchConfig("* * * * *", source, sink, new ArrayList<ETLStage>());
+    ETLBatchConfig etlConfig = ETLBatchConfig.builder("* * * * *")
+      .setSource(source)
+      .addSink(sink)
```
Comment (gsps1) [inline mongodb-plugins/src/test/java/co/cask/hydrator/plugin/test/MongoDBTest.java:237]: you didn't add connection here

File: mongodb-plugins/src/test/java/co/cask/hydrator/plugin/test/MongoDBTest.java [hunk 2]
```diff
@@ -270,7 +277,10 @@ public void testMongoToMongo() throws Exception {
       .put(MongoDBBatchSink.Properties.CONNECTION_STRING,
            String.format("mongodb://localhost:%d/%s.%s",
                          mongoPort, MONGO_DB, MONGO_SINK_COLLECTIONS)).build()));
-    ETLBatchConfig etlConfig = new ETLBatchConfig("* * * * *", source, sink, new ArrayList<ETLStage>());
+    ETLBatchConfig etlConfig = ETLBatchConfig.builder("* * * * *")
+      .setSource(source)
+      .addSink(sink)
```
Comment (gsps1) [inline mongodb-plugins/src/test/java/co/cask/hydrator/plugin/test/MongoDBTest.java:283]: same here.

**Comment by gsps1** [comment]
fix and merge. lgtm

File: mongodb-plugins/src/test/java/co/cask/hydrator/plugin/test/MongoDBTest.java (hunk 1 above)
Comment (albertshau) [inline mongodb-plugins/src/test/java/co/cask/hydrator/plugin/test/MongoDBTest.java:237]: good catch, fixed.

**Pull request merged**